runtime-extension-model/src/main/resources/META-INF/mule-operation.xsd

```diff
@@ -283,4 +283,16 @@
         </xsd:restriction>
     </xsd:simpleType>
 
+    <xsd:simpleType name="visibilityType">
+        <xsd:annotation>
+            <xsd:documentation>
+                Whether the operation is public and should be usable by third party components
+            </xsd:documentation>
+        </xsd:annotation>
+        <xsd:restriction base="xsd:string">
+            <xsd:enumeration value="public"/>
```

Comment:
should this be upper case? Don't we usually use upper for enums? I'm only asking, not sure what the consistent thing is and don't have a strong opinion on this

Reply:
In this same xsd we don't follow any convention: 
```
<xsd:simpleType name="basicTypes">
        <xsd:annotation>
            <xsd:documentation>
                The basic types for input parameters or component's output
            </xsd:documentation>
        </xsd:annotation>
        <xsd:restriction base="xsd:string">
            <!--            TODO: HOW TO REPRESENT ARRAYS?-->
            <xsd:enumeration value="string"/>
            <xsd:enumeration value="number"/>
            <xsd:enumeration value="binary"/>
            <xsd:enumeration value="date"/>
            <xsd:enumeration value="datetime"/>
            <xsd:enumeration value="time"/>
            <xsd:enumeration value="boolean"/>
            <xsd:enumeration value="binary"/>
        </xsd:restriction>
    </xsd:simpleType>
```
And: 
```
<xsd:restriction base="xsd:string">
        <xsd:enumeration value="SUPPORTED"/>
        <xsd:enumeration value="NOT_SUPPORTED"/>
        <xsd:enumeration value="REQUIRED"/>
</xsd:restriction>
```
I just added like this because it sounded more familiar to me to see then in lower case, but we can change it